shared/constants/search.js

```diff
@@ -78,6 +78,18 @@ export function searchResultToAssertion (r: SearchResult): string {
   return r.username
 }
 
+export function usernameToSearchResult (username: string): SearchResult {
```

Comment:
Here we make a fake result, could maybe set the TODOs but kinda a bunch of work. deferring